Number_types/include/CGAL/Mpzf.h

```diff
@@ -774,7 +776,11 @@ struct Mpzf {
     return res;
   }
 
+#ifndef CGAL_MPZF_DIVISION_OPERATOR
+  friend Mpzf division(Mpzf const&a, Mpzf const&b){
```

Comment:
Wouldn't it make sense to have a base class without divsion (or the other way round)?

Reply:
You would need to redefine all the operations in the derived class so they return the right type, that seems ugly. And I don't clearly see how it would help.